### PR TITLE
Don't create build dir in docker build

### PR DESCRIPTION
**Pull request by kelvinabrokwa**
Since there is only one build artifact (the `.hex` file) we can just place it in the root directory

re: https://github.com/tribesat/firmware/issues/2

### CODE REVIEW

**Review by eriknstevenson — COMMENTED** [review]
I think this is fine. If possible, I'd like to get a couple other minor things changed as well. I've opened up #5.

**Comment by kelvinabrokwa** [comment]
@narrative sure I'll take a look at this. Can you please give this an official review?

**Comment by eriknstevenson** [comment]
Since #5 is piggy backing off of this branch, I was going to wait until it is either approved/denied that way we can merge it all in at once.

**Review by eriknstevenson — APPROVED** [review]

**Pull request merged**
